src/basho_bench_worker_sup.erl

```diff
@@ -60,8 +60,28 @@ init([]) ->
     %% specs for each
 
     basho_bench_profiler:maybe_start_profiler(basho_bench_config:get(enable_profiler, false)),
-    Workers = worker_specs(basho_bench_config:get(concurrent), []),
-    {ok, {{one_for_one, 5, 10}, Workers}}.
+
+    Workers = basho_bench_config:get(workers, []),
+    WorkerTypes = basho_bench_config:get(worker_types, []),
```

Comment:
No. Right idea but the wrong spot. We want to tolerate both Workers and WorkerTypes being undefined to support the traditional case. But if Workers is defined and there isn't a matching WorkerTypes entry we want THAT to blow up, below in line 77 so I am changing that to be where it fails. That would also cover WorkerTypes being undefined. 

                                      Conf = proplists:get_value(WT, WorkerTypes),